README.md

```diff
@@ -1,75 +1,67 @@
 # fedcloud-catchall-operations
 
-Operation of fedcloud integration components for selected providers. 
-This is a helm chart that will deploy the catchall operations for
-the EGI federated cloud.
+Operation of fedcloud integration components for selected providers.
+This is a set of docker containers and an ansible role to operate the
+federation components of the EGI Cloud
 
 ## What this does?
```

Comment:
Punctuation in title may not be ideal, it's often discouraged (both by the linters and also in some guidelines)